compiler/optimizer/RedundantAsyncCheckRemoval.cpp

```diff
@@ -420,13 +420,22 @@ bool TR_RedundantAsyncCheckRemoval::callDoesAnImplicitAsyncCheck(TR::Node *callN
        (symbol->getRecognizedMethod()==TR::java_lang_Integer_rotateLeft) ||
        (symbol->getRecognizedMethod()==TR::java_lang_Long_rotateLeft) ||
        (symbol->getRecognizedMethod()==TR::java_lang_Integer_rotateRight) ||
-       (symbol->getRecognizedMethod()==TR::java_lang_Long_rotateRight) ||
```

Comment:
If this code is shared between Java8 (where the methods are native) and Java9 (where they are not native as you mentioned in the abstract), how does this change work in both cases ?